tests/robot-tests/tests/admin_and_public/bau/legacy_releases.robot

```diff
@@ -33,36 +38,86 @@ Create legacy release
     user clicks button    OK
     user waits until modal is not visible    Create legacy release
     user waits until h1 is visible    Create legacy release
-    user enters text into element    id:legacyReleaseForm-description    Test collection
+    user enters text into element    id:legacyReleaseForm-description    ${DESCRIPTION}
     user enters text into element    id:legacyReleaseForm-url    http://test.com
     user clicks button    Save legacy release
 
 Validate created legacy release
     user waits until h1 is visible    Manage publication
     user checks element count is x    css:tbody tr    1
     user checks results table cell contains    1    1    1
-    user checks results table cell contains    1    2    Test collection
+    user checks results table cell contains    1    2    ${DESCRIPTION}
     user checks results table cell contains    1    3    http://test.com
 
+Navigate to admin dashboard to create new release
+    user navigates to admin dashboard    Bau1
+
+Navigate to release
```

Comment:
Could just combine these two into `Navigate to release`. Also, the first `Navigate to admin dashboard to create new release` doesn't make sense anyway as we've already created the release.

Reply:
I don't consider this a huge deal, but I would keep a reference to the admin site here - `Navigate to release` itself is ambiguous as it isn't clear if it's the release on admin or public.

My preference would for test case names to always indicate when the tests are switching from admin to public or vice versa.

Reply:
Could just call it `Navigate to release in admin`?